docs/usage/from-a-terminal.rst

```diff
@@ -1129,7 +1134,7 @@ Global overview
                                 the output directory from.
                                 Configured value: '/XXXX/XXXX/XXXXX'
         --unified-results     Activates or disables the generation of the unified
-                                results file instead of the splitted one.
+                                results file instead of the split one.
```

Comment:
Please also apply the change at: 
https://github.com/funilrys/PyFunceble/blob/0d5e5450473aff6712277d8dd0bbc240e2f953f1/PyFunceble/cli/entry_points/pyfunceble/cli.py#L686-L687

Reply:
TODO

Find a dictionary...

Reply:
According to wiktionary.org it would probably be better to actually replace split with `divided`

https://en.wiktionary.org/wiki/divided#English

https://en.wiktionary.org/wiki/split#Verb

![image](https://user-images.githubusercontent.com/44526987/104138128-7e145080-53a2-11eb-81cb-bd329f521cb6.png)

![image](https://user-images.githubusercontent.com/44526987/104138138-88cee580-53a2-11eb-95f9-d230d3d58f9f.png)

I'll make these changes and post a new edition

Reply:
Are you moving the project on this one